data/json/itemgroups/Clothing_Gear/gear_civilian.json

```diff
@@ -140,9 +128,6 @@
       { "group": "toy_robot_kit", "prob": 1 },
       { "group": "toy_engine_kit", "prob": 1 },
       { "item": "air_launcher", "prob": 1 },
-      [ "breadboard", 30 ],
-      { "item": "voltmeter", "prob": 20, "charges": [ 0, 100 ] },
-      { "item": "multimeter", "prob": 20, "charges": [ 0, 100 ] },
```

Comment:
These are probably significantly on the high end probability wise, but I could see them in a kit like the other items just above them.

Reply:
I have answered that concern up above in a discussion. I don't believe a primary school child should randomly have voltmeters in their pocket. This is possible but the odds are so low we may as well just add any item they may or may not have carried at that point. To be honest, their itemgroup is already a mess. While with adults the items are mostly stuff one can reasonable have in their pockets, the children just have EVERYTHING that feels vaguely child-like.

Reply:
Ah, my bad for retreading the discussion.